server/controllers/appointmentsController.ts

```diff
@@ -35,6 +35,13 @@ export default class AppointmentsController {
     }
   }
 
+  updateProjectDetails(): RequestHandler {
```

Comment:
I've gone a step further and created a separate controller for attendance outcomes given the example you shared. Happy for this to go in as is, but just flagging.